zipline/testing/fixtures.py

```diff
@@ -164,6 +166,39 @@ def add_instance_callback(self, callback):
         return self._instance_teardown_stack.callback(callback)
 
 
+def alias(attr_name):
+    """Make a fixture attribute an alias of another fixture's attribute by
+    default.
+
+    Parameters
+    ----------
+    attr_name : str
+        The name of the attribute to alias.
+
+    Returns
+    -------
+    p : classproperty
+        A class property that does the propert aliasing.
+    """
+    return classproperty(flip(getattr, attr_name))
```

Comment:
A usage example in the docstring here would be nice for the non-functionally-inclined.